src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -98,8 +103,14 @@ internal sealed partial class LocalRewriter : BoundTreeRewriterWithStackGuard
 
                 var instrumenter = Instrumenter.NoOp;
 
+                if (instrumentation.Kinds.Contains(InstrumentationKind.LocalStateTracing) &&
```

Comment:
>Kinds.Contains(InstrumentationKind.LocalStateTracing) [](http://example.com/codeflow?start=36&length=53)

This call appears multiple times. Would it make sense to add a `bool IncludesLocalStateTracing { get; }` property to `MethodInstrumentation`?

Reply:
idk, doesn't seem like helpers like that would add that much benefit.